starter-code/routes/passportRouter.js

```diff
@@ -5,16 +5,79 @@ const User           = require("../models/user");
 // Bcrypt to encrypt passwords
 const bcrypt         = require("bcrypt");
 const bcryptSalt     = 10;
-const ensureLogin = require("connect-ensure-login");
-const passport      = require("passport");
+const ensureLogin    = require("connect-ensure-login");
+const passport       = require("passport");
 
 
 
-router.get("/private-page", ensureLogin.ensureLoggedIn(), (req, res) => {
-  res.render("passport/private", { user: req.user });
+router.get("/private", ensureLogin.ensureLoggedIn(), (req, res) => {
+  res.render("./passport/private", { user: req.user });
```

Comment:
for the next time remove the dot from the view rendering comand. it works but it shouldn't and it's wrong